README.adoc

```diff
@@ -6,6 +6,37 @@ image:https://codecov.io/gh/codeready-toolchain/toolchain-common/branch/master/g
 
 This repo is for controllers, libs, scripts, make files, etc to be shared between host and member operators.
 
+== Setting Up and Connecting Host and Member Operators
+
+In a new terminal, navigate to codeready-toolchain/host-operator directory and execute the following commands
+```
+$ minishift start --profile toolchain
+$ make up-local
```

Comment:
ideally, we should run something like `make setup` so we don't have to `ctrl+c` afterwards

Reply:
@tinakurian could you please address this ^^^ comment? Starting and killing the operator just for the sake of deploying resources is not the best way of showing the functionality.

Reply:
@xcoulon @MatousJobanek This is definitely something we could add. The task was really to document how we can get both host and member running and talking to each other. I can create a separate task to create a script in order to address this - this will also help avoid scope creep

Reply:
The main task is to improve the user experience for ppl that are new to the project. If there is a room for improvement to make the experience better, then we should do it. In addition, the targets should have been added in host/member projects, so you can do it in parallel.

Reply:
@MatousJobanek I'm not sure we are talking about the same story? This was the story which i created related to this work: https://jira.coreos.com/browse/CRT-178. The goal of this story was really for me to learn how to get both host and member running and talking to each other and also to document the process. Again, not apposed to improving the process, just think it should be done as part of a different story/task. I didn't have this in mind when i wrote the story and allocated points to it.

Reply:
OK, if you don't want to change the makefile targets (as part of this task) then I do respect your decision, but please, don't use the `make up-local` for deploying the resources. `make up-local` is a target to be used for running operator locally, not for deploying resources. instead of this, you can use a combination of targets:
`make login-as-admin create-namespace deploy-rbac deploy-crd`
but this is already documented in both operator and member projects:
https://github.com/codeready-toolchain/host-operator#development
https://github.com/codeready-toolchain/member-operator#development

Reply:
@MatousJobanek Done! This is much clearer - thanks for the suggestion!